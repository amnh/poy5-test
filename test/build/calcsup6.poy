(* calculate_support with bremer build of_file*)
(*terminals reduced to random 10 to expidite test*)
read ("2.fas")
select (terminals, _random:10)
set (seed:1)
build(1)
swap()
select()
calculate_support(bremer, build(of_file:"2.fas.2.tree"))
report(supports:bremer)