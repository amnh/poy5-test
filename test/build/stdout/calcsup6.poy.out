Bremer Supports:
Support tree:
(Americhernus,Thelichoris)inf
